.github/workflows/build-&-publish-docker-image.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       run: |
         if [[ "$PUSH_LATEST" == "yes" ]] && [[ "$BRANCH" == "master" ]]; then
           docker push $SHARDER_REGISTRY:latest
-        elif [[ "$BRANCH" == "staging" ]]; then
+        elif [[ "$PUSH_LATEST" == "yes" ]] && [[ "$BRANCH" == "staging" ]]; then
```

Comment:
see my slack reply, we should always be pushing latest as latest image should reflect latest commit